### PR TITLE
:tada: initial commit api contracts and dependency

### PR DESCRIPTION
**Pull request by brohit-97**

### CODE REVIEW

File: .idea/jpa-buddy.xml
Comment (ajeetcloud) [inline .idea/jpa-buddy.xml:1]: remove unrequired files from this PR

File: grievance-management-service/src/main/java/com/scaler/grievancemanagementservice/controllers/GrievanceOperations.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
+package com.scaler.grievancemanagementservice.controllers;
+
+import com.scaler.grievancemanagementservice.models.Grievance;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.Optional;
+
+@RequestMapping("/api/v1/grievances")
```
Comment (ajeetcloud) [inline grievance-management-service/src/main/java/com/scaler/grievancemanagementservice/controllers/GrievanceOperations.java]: /grievances

File: .idea/jpa-buddy.xml
Comment (brohit-97) [reply .idea/jpa-buddy.xml:1]: Done

File: grievance-management-service/src/main/java/com/scaler/grievancemanagementservice/controllers/GrievanceOperations.java (hunk 1 above)
Comment (brohit-97) [reply grievance-management-service/src/main/java/com/scaler/grievancemanagementservice/controllers/GrievanceOperations.java]: Ok

**Review by ajeetcloud — APPROVED** [review]
LGTM

**Pull request merged**